pages/docs/components/layout/container.mdx

```diff
@@ -43,14 +39,30 @@ To contain any piece of content, wrap it in the `Container` component.
 
 ## Container Size
 
-To set the size of a Container, use the `maxW` attribute.
+By default, the `Container` component sets the `maxWidth` of the content to 60
+characters (`60ch`) but you can customize this by passing custom `maxWidth`
+values or changing the size tokens defined in
+[theme.sizes](https://chakra-ui.com/docs/styled-system/theming/theme#sizes).
+
+> - About the default value: The `ch` unit is a relative unit defined by the
+>   rendered typeface's "0" character width. This width varies by the shape and
+>   style of the font.
+> - If you are curious about the reason for this default value of `60`
+>   characters, consider this explanation about
+>   [line length](https://betterwebtype.com/articles/2019/06/16/5-keys-to-accessible-web-typography/#line-length)
+>   from Better Web Type.
 
 ```jsx
 <VStack>
-  <Container maxW='container.xl'>Extra-Large Container</Container>
-  <Container maxW='container.lg'>Large Container</Container>
-  <Container maxW='container.md'>Medium Container</Container>
-  <Container maxW='container.sm'>Small Container</Container>
+  <Container maxW='md' bg='blue.600' color='white'>
+    "md" Container
+  </Container>
+  <Container maxW='550px' bg='purple.600' color='white'>
+    "550px" Container
+  </Container>
+  <Container maxW='container.sm' bg='green.400' color='#262626'>
+    "container.sm" Container
+  </Container>
```

Comment:
I'd also leave the old example as it was, as it was meant to show the different default sizes that we have for `container`. I'd add an example with custom `maxW` after mentioning that it's a way to style it.
The first paragraph could be about `maxW` then example and the second with sizes from theme with the old example.

Reply:
Along with adding in a background color to each `Container` to be able to better see it's width, I was thinking that the original example didn't truly contribute anything, as some of the sizes from the `container` token are greater than the preview window's widest point (around a `767px` screen size), hence linking to the style props page via the `theme.sizes.container` text in the details or listing the container token sizes instead, and simplifying the example.

Thoughts?

Reply:
Convinced :D Is the container.md working in the example ? At least to add a second one ?

Reply:
`container.md` is wider than the other new examples with a max-width of `768`, so it will always show as maxed-out in the preview (I know, just shy! 🤣 The layout shift of the page happens at the 768 breakpoint). But I suppose that would be fine as an additional example. 😃

Reply:
I don't think it's possible to showcase the container widths within the docsite itself. One thing we can do it setup a CodeSandbox example with the different widths and share the full page example (not the csb IDE link). That way the examples will be in full width of the screen and the users will be able to see the differences.

Reply:
@nikolovlazar in that case I thin I'd just go with what @TylerAPfledderer did for now. And if we see people struggle we can add the csb example?

Reply:
sounds good!